release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -134,6 +134,12 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 
 For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-bios_ipi-install-installation-workflow[Configuring the BIOS] and xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-raid_ipi-install-installation-workflow[Configuring the RAID].
 
+[id="ocp-4-11-capabilities"]
+==== Additional customization when installing {product-title}
+{product-title} 4.11 allows you to disable the installation of the `baremetal` and `marketplace` Operators, and the `samples` content that is stored in the `openshift` namespace. You can disable these features by setting the `baselineCapabilitySet` and `additionalEnabledCapabilities` parameters in the `install-config.yaml` configuration file prior to installation. If you disable any of these capabilities during the installation, you can enable them after the cluster is installed. Once a capability has been enabled, it cannot be disabled again.
```

Comment:
I would use `openshift-samples` to match [the API slug][1].

[1]: https://github.com/openshift/api/blob/1d69d7990d9298bf056c8caf05c75bb59a3f441e/config/v1/types_cluster_version.go#L237

Reply:
Incorporated this feedback, thank you